test/client/fixtures/sandboxes/cookie-test.js

```diff
@@ -84,7 +84,7 @@ test('get/set', function () {
 asyncTest('path validation', function () {
     var iframe = document.createElement('iframe');
 
-    iframe.src = UrlUtil.getProxyUrl('/data/cookie-sandbox/validation.html');
+    iframe.src = UrlUtil.getProxyUrl(window.QUnitGlobals.getResourceUrl('../../data/cookie-sandbox/validation.html', 'cookie-sandbox/validation.html'));
```

Comment:
So, the second parameter is the URL alias, right? But what is the default URL then (without specifying this alias) ?